package.json

```diff
@@ -16,21 +16,18 @@
   "sideEffects": false,
   "scripts": {
     "build": "make build",
-    "build-docs": "build-storybook && npm run build-gatsby && npm run move-gatsby",
-    "build-storybook": "build-storybook",
+    "build-docs": "npm run build-gatsby && npm run move-gatsby",
```

Comment:
"deploy" is missing in these scripts.

If you remove "move-gatsby" it could be something like
`"deploy": "gh-pages -d ./www/public"`